tests/e2e/instrumentation-sdk/00-install-instrumentation.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: sdk-only
+spec:
+  env:
+    - name: OTEL_TRACES_EXPORTER
```

Comment:
I would prefer to remove all this OTEL  env vars rely on the env vars derived from the `spec.`